dkms.in

```diff
@@ -2261,7 +2261,7 @@ autoinstall() {
                     installed_modules[${#installed_modules[@]}]="$m"
                     install_count=$(($install_count +1))
                 else
-                    failed_modules=[${#failed_modules[@]}]="$m"
+                    failed_modules[${#failed_modules[@]}]="$m($?)"
```

Comment:
Nicely found. Out of curiosity, how did you spot it - code review or some tool/bash warning option?

Reply:
when I printed the array, it contained garbage ;-)